### PR TITLE
Issue #606: Extended Eclipse user guide for newbies of Checkstyle.

### PR DESCRIPTION
**Pull request by MEZk**

### CODE REVIEW

**Comment by romani** [comment]
1) can we make new pictures less wide ? there too much empty space on them. It would be good to resize windows to be compact.

2) unrelated , but please do me favour, why I see a that.
Chapters order:
Debug
Organize Imports

make as:
Organize Imports
Debug

in separate PR or commit.
Reason: user prepare IDE first, and then do debugging and fixing a code.

File: src/xdocs/eclipse.xml [hunk 1]
```diff
@@ -26,6 +26,13 @@
       </macro>
     </section>
 
+    <section name="Download Eclipse">
+      <p>
+        Please visit Eclipse IDE <a href="https://eclipse.org/"> official web site</a>,
+        download and install the lates version of Eclipse IDE for Java Developers.
```
Comment (mkordas) [inline src/xdocs/eclipse.xml]: typo: `lates`

**Comment by mkordas** [comment]
Minor: I don't think that commit message and PR title requires dot at the end

**Comment by MEZk** [comment]
@romani 
1) Done.
2) Done as separate commit.

@mkordas 
Thank you again. 
>Minor: I don't think that commit message and PR title requires dot at the end

Ok.

**Comment by romani** [comment]
merged as FF

**Pull request closed without merging**